CHANGELOG.md

```diff
@@ -25,6 +25,8 @@ Main (unreleased)
 
 - Integrations: Introduce the `mssql` integration. (@binaryfissiongames)
 
+- Integrations: Introduce `cloudwatch metrics` integration. (@thepalbi)
```

Comment:
@marctc just fixed it. Terrible mistake trying to merge conflicts with the gh ui 🤦